client-src/js-src/cs-client.js

```diff
@@ -646,6 +646,14 @@ export class ChromeStatusClient {
     return this.doGet(`/features/${featureId}/process`);
   }
 
+  // Intents API
```

Comment:
Once you have OpenAPI endpoints defined (in your other PR), I think that you can call generated code rather than adding code to cs-client.

Reply:
I think you're right, and this is something that I will implement in a subsequent PR once moving these files to TypeScript. 🙂